src/Symfony/Component/VarExporter/Tests/LazyGhostTraitTest.php

```diff
@@ -503,6 +503,14 @@ public function testPropertyHooks()
         $object->backed = 345;
         $this->assertTrue($initialized);
         $this->assertSame(345, $object->backed);
+
+        $initialized = false;
+        $object = $this->createLazyGhost(Hooked::class, function ($instance) use (&$initialized) {
```

Comment:
I suggest making a separate test instead. This way, the test name can describe what is actually tested here (and it also provide better feedback about what is failing in case of failure, as well as ensuring this case is tested even if the previous one fails)